src/test/java/RecipeTest.java

```diff
@@ -19,23 +19,29 @@
 import org.junit.Test;
 import java.util.LinkedList;
 import java.util.List;
+import java.util.HashSet;
+import java.util.Set;
 import java.util.Arrays;
 
 @RunWith(JUnit4.class)
 public final class RecipeTest {
 
   private static final String NAME = "Grilled Cheese";
   private static final String DESCRIPTION = "It's literally just melted cheese on bread";
+  private static final Set<String> TAGS = new HashSet<>(Arrays.asList("grilledcheese", "quick"));
+  private static final Set<String> INGREDIENTS = new HashSet<>(Arrays.asList("bread", "cheese", "butter"));
   private static final List<Step> STEPS = Arrays.asList(
       new Step("Toast the bread for 2 minutes"),
       new Step("Melt the cheese"),
       new Step("Put the cheese in the bread")
   );
+  private static final Recipe spinOff = new Recipe("Spicy grilled cheese", "it's hot", TAGS, INGREDIENTS, STEPS);
```

Comment:
Spinoff need not be a global variable. It is required only in few tests, so please initialize only in those tests.